test/runtime/fqdn.go

```diff
@@ -1013,6 +1014,27 @@ var _ = Describe("RuntimeFQDNPolicies", func() {
 		invalidURL := "http://world1.outside.test"
 		invalidIP := outsideIps[OutsideHttpd1]
 
+		// Apply a CIDR policy with some link local IPs first so that the DNS identities
+		// will not allocate from the beginning of the local identity range
```

Comment:
And why do we want to avoid that?

Reply:
So that we'd get different identities allocated after restart (where we do not import this CIDR policy) if the identity restoration did not work. Will add this to the comment.